core/src/main/java/nl/nn/adapterframework/jdbc/migration/LiquibaseMigrator.java

```diff
@@ -96,6 +98,14 @@ private Liquibase createMigrator(Resource resource) throws SQLException, Liquiba
 		ResourceAccessor resourceAccessor = new LiquibaseResourceAccessor(resource);
 		DatabaseConnection connection = getDatabaseConnection();
 
+		Map<String, Object> scopeValues = new HashMap<>();
+		scopeValues.put(Scope.Attr.resourceAccessor.name(), resourceAccessor);
+		try {
+			Scope.enter(scopeValues);
+		} catch (Exception e) {
+			throw new LiquibaseException(e);
+		}
+
```

Comment:
Maar als je nu meerdere configuraties hebt met liquibase, gaat hij steeds een nieuwe scope in?
Wat is nou precies het probleem? Liquibase maat gebruik van de `java.util.ServiceLoader`. Kunnen we niet gewoon dingen uitzetten?

Reply:
Ik begrijp uit de liquibase documentatie dat hij 'gewoon' uit scope gaat als hij uit scope gaat.

Het probleem is dat de changelog parser niet de resourceAccessor gebruikt die hij meekrijgt, maar hem uit de scope haalt. Met deze wijziging staat de goede resourceAccessor in de scope.